packages/components/src/components/ScalarFloating/ScalarFloating.vue

```diff
@@ -72,15 +72,19 @@ const { floatingStyles, middlewareData } = useFloating(targetRef, floatingRef, {
     :class="{ contents: !!$slots.default }">
     <slot />
   </div>
-  <div
-    ref="floatingRef"
-    class="relative z-context"
-    :style="floatingStyles"
-    v-bind="$attrs">
-    <slot
-      :data="middlewareData"
-      :height="targetHeight"
-      name="floating"
-      :width="targetWidth" />
-  </div>
+  <Teleport
```

Comment:
I think we should make the teleport opt-in since it can cause some accessibility and styling issues. You can use the [`disabled` prop](https://vuejs.org/guide/built-ins/teleport#disabling-teleport) to have it turned off by default but then have a prop to turn it on.

Reply:
updated accordingly, using `teleport` with enable it from now on